.github/workflows/kind-e2e-containerd.yaml

```diff
@@ -34,7 +37,7 @@ jobs:
             kind-version: v0.14.0
             kind-image-sha: sha256:b1fa224cc6c7ff32455e0b1fd9cbfd3d3bc87ecaa8fcb06961ed1afb3db0f9ae
 
-          - k8s-version: v1.24.0
+          - k8s-version: v1.24.2
             kind-version: v0.14.0
             kind-image-sha: sha256:0866296e693efe1fed79d5e6c7af8df71fc73ae45e3679af05342239cdc5bc8e
```

Comment:
SHA needs to be updated too: `sha256:1f0cee2282f43150b52dc7933183ed96abdcfc8d293f30ec07082495874876f1`